common-service-core/src/main/java/org/zowe/apiml/message/api/ApiMessage.java

```diff
@@ -36,11 +36,13 @@
     private String messageSource;
 
 
-    public ApiMessage(String messageKey, MessageType messageType, String messageNumber, String messageContent) {
+    public ApiMessage(String messageKey, MessageType messageType, String messageNumber, String messageContent, String messageAction, String messageReason) {
```

Comment:
Any specific reason why we have different order here and in Message class?

Reply:
Yes that was my mistake, I fixed it and forgot to push